contact.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/grid.css">
+    <title>Document</title>
+</head>
+
+<div class="wrapper">
+    <div class="grid-container">
+        <header class="grid-header">
+        </header>
+
+        <nav class="grid-top-nav">
+            <ul id="grid-nav-ul">
+                <li id="home"><a href="index.html">Home</a></li>
+                <li id="about"><a href="about.html">About</a></li>
+                <li id="portfolio"><a href="portfolio.html">Portfolio</a></li>
+                <li id="blog"><a href="code-journal.html">Blog</a></li>
+                <li id="contact"><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+
+        <main class="grid-main" id="hero">
+            <div class="contact-form-container">
+                <form>
+
+                    <label for="first-name">First Name</label>
```

Comment:
The `for` attribute must match the `id` of a form control (an input or something).